script/delete_weird_company.rb

```diff
@@ -0,0 +1,32 @@
+require File.expand_path('../../config/environment', __FILE__)
+
+weird_company_wql = {
+  'not' => { 'type' => 'Company' },
+  'left_plus' => [
+    { 'type' => 'Metric' },
+    { 'right_plus' => [{ 'type' => 'Year' }, { 'type' => 'Metric Value' }] }
+  ]
+}
+Card::Auth.as_bot
+weird_companies = Card.search weird_company_wql
+weird_companies.each do |company|
+  related_cards_wql = { left: { left: { type_id: Card::MetricID },
+                                right: company.name } }
+  related_cards = Card.search related_cards_wql
+  related_cards.each do |related_card|
+    if (value_card = Card[related_card.name + '+value'])
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.